techlahoma/200ok-site
# Add atlas school sponsor

To test: 

- Load page
- Scroll down to sponsors
- Expected: Atlas logo should appear with text about its sponsorship level beneath it
